editor/converter.cpp

```diff
@@ -0,0 +1,3042 @@
+	{ "indianred", "INDIAN_RED" },
+	{ "indigo", "INDIGO" },
+	{ "ivory", "IVORY" },
+	{ "khaki", "KHAKI" },
+	{ "lavender", "LAVENDER" },
+	{ "lavenderblush", "LAVENDER_BLUSH" },
+	{ "lawngreen", "LAWN_GREEN" },
+	{ "lemonchiffon", "LEMON_CHIFFON" },
+	{ "lightblue", "LIGHT_BLUE" },
+	{ "lightcoral", "LIGHT_CORAL" },
+	{ "lightcyan", "LIGHT_CYAN" },
+	{ "lightgoldenrod", "LIGHT_GOLDENROD" },
+	{ "lightgray", "LIGHT_GRAY" },
+	{ "lightgreen", "LIGHT_GREEN" },
+	{ "lightpink", "LIGHT_PINK" },
+	{ "lightsalmon", "LIGHT_SALMON" },
+	{ "lightseagreen", "LIGHT_SEA_GREEN" },
+	{ "lightskyblue", "LIGHT_SKY_BLUE" },
+	{ "lightslategray", "LIGHT_SLATE_GRAY" },
+	{ "lightsteelblue", "LIGHT_STEEL_BLUE" },
+	{ "lightyellow", "LIGHT_YELLOW" },
+	{ "lime", "LIME" },
+	{ "limegreen", "LIME_GREEN" },
+	{ "linen", "LINEN" },
+	{ "magenta", "MAGENTA" },
+	{ "maroon", "MAROON" },
+	{ "mediumaquamarine", "MEDIUM_AQUAMARINE" },
+	{ "mediumblue", "MEDIUM_BLUE" },
+	{ "mediumorchid", "MEDIUM_ORCHID" },
+	{ "mediumpurple", "MEDIUM_PURPLE" },
+	{ "mediumseagreen", "MEDIUM_SEA_GREEN" },
+	{ "mediumslateblue", "MEDIUM_SLATE_BLUE" },
+	{ "mediumspringgreen", "MEDIUM_SPRING_GREEN" },
+	{ "mediumturquoise", "MEDIUM_TURQUOISE" },
+	{ "mediumvioletred", "MEDIUM_VIOLET_RED" },
+	{ "midnightblue", "MIDNIGHT_BLUE" },
+	{ "mintcream", "MINT_CREAM" },
+	{ "mistyrose", "MISTY_ROSE" },
+	{ "moccasin", "MOCCASIN" },
+	{ "navajowhite", "NAVAJO_WHITE" },
+	{ "navyblue", "NAVY_BLUE" },
+	{ "oldlace", "OLD_LACE" },
+	{ "olive", "OLIVE" },
+	{ "olivedrab", "OLIVE_DRAB" },
+	{ "orange", "ORANGE" },
+	{ "orangered", "ORANGE_RED" },
+	{ "orchid", "ORCHID" },
+	{ "palegoldenrod", "PALE_GOLDENROD" },
+	{ "palegreen", "PALE_GREEN" },
+	{ "paleturquoise", "PALE_TURQUOISE" },
+	{ "palevioletred", "PALE_VIOLET_RED" },
+	{ "papayawhip", "PAPAYA_WHIP" },
+	{ "peachpuff", "PEACH_PUFF" },
+	{ "peru", "PERU" },
+	{ "pink", "PINK" },
+	{ "plum", "PLUM" },
+	{ "powderblue", "POWDER_BLUE" },
+	{ "purple", "PURPLE" },
+	{ "rebeccapurple", "REBECCA_PURPLE" },
+	{ "red", "RED" },
+	{ "rosybrown", "ROSY_BROWN" },
+	{ "royalblue", "ROYAL_BLUE" },
+	{ "saddlebrown", "SADDLE_BROWN" },
+	{ "salmon", "SALMON" },
+	{ "sandybrown", "SANDY_BROWN" },
+	{ "seagreen", "SEA_GREEN" },
+	{ "seashell", "SEASHELL" },
+	{ "sienna", "SIENNA" },
+	{ "silver", "SILVER" },
+	{ "skyblue", "SKY_BLUE" },
+	{ "slateblue", "SLATE_BLUE" },
+	{ "slategray", "SLATE_GRAY" },
+	{ "snow", "SNOW" },
+	{ "springgreen", "SPRING_GREEN" },
+	{ "steelblue", "STEEL_BLUE" },
+	{ "tan", "TAN" },
+	{ "teal", "TEAL" },
+	{ "thistle", "THISTLE" },
+	{ "tomato", "TOMATO" },
+	{ "transparent", "TRANSPARENT" },
+	{ "turquoise", "TURQUOISE" },
+	{ "violet", "VIOLET" },
+	{ "webgray", "WEB_GRAY" },
+	{ "webgreen", "WEB_GREEN" },
+	{ "webmaroon", "WEB_MAROON" },
+	{ "webpurple", "WEB_PURPLE" },
+	{ "wheat", "WHEAT" },
+	{ "white", "WHITE" },
+	{ "whitesmoke", "WHITE_SMOKE" },
+	{ "yellow", "YELLOW" },
+	{ "yellowgreen", "YELLOW_GREEN " },
```

Comment:
```suggestion
	{ "yellowgreen", "YELLOW_GREEN" },
```

Reply:
This was intentional.
I want to see if CI will fail